make.py

```diff
@@ -61,15 +61,14 @@ def _print_usage():
 
 
 if __name__ == '__main__':
-    avail_cmds = dict(clean=clean, dist=dist, docs=docs)
+    avail_cmds = {clean=clean, dist=dist, docs=docs}
```

Comment:
My apologies for not flagging this in the previous review.  To my knowledge, this is not valid syntax in any current version of Python.

Reply:
I don't think the tiny performance benefit is worth it. It's just not pythonic.

Reply:
As pushfoo pointed out, your proposed change is not valid Python syntax. 

I don't agree that the original code is non-Pythonic.